apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -86,6 +88,7 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
                 ...router.query,
                 date: slot.time.format(),
                 type: eventTypeId,
+                slug: eventTypeSlug,
```

Comment:
This is required to identify the event type of the dynamic booking because the eventTypeId is 0 for dynamic booking events (pulled from a file, not the database)
This would only be looked at when eventTypeId is 0 which means it is for a dynamic group booking